.travis.yml

```diff
@@ -1,7 +1,9 @@
 sudo: true
 dist: bionic
 language: java
-jdk: openjdk8
+jdk:
+  - openjdk8
+  - openjdk11
```

Comment:
I don't think this does what you want it to do.
With this, you seem to have triggered a [completely new build stage](https://travis-ci.com/github/opencast/opencast/builds/196623948) running `mvn test`. The test stage [did not exist before](https://travis-ci.com/github/opencast/opencast/builds/196673042). This is unnecessary and makes the tests way slower. You build twice in row after all.

I think you just want the build step to be executed with both JDKs. Not sure if it works, but I would try:

```yaml
    # Make a build excluding the assembly steps
    - stage: build
      env: name=build
      jdk:
        - openjdk8
        - openjdk11
      script:
```